examples/by_feature/multi_node_metrics.py

```diff
@@ -0,0 +1,215 @@
+# coding=utf-8
```

Comment:
Should be named `multi_process_metrics` as the problem arises as soon as you have several processes, not several nodes.